src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -299,7 +313,7 @@ export default function connectRefinementList(renderFn, unmountFn = noop) {
           showMore ? showMoreLimit : limit
         );
 
-        return widgetConfiguration;
+        return new algoliasearchHelper.SearchParameters(widgetConfiguration);
```

Comment:
You can use the argument provided by the `getConfiguration` function.

Reply:
d51f2c670bcef53e4f28ad1f8c0e7a58d7658ba4